client/app/dashboard/appSubmit.js

```diff
@@ -15,8 +15,16 @@ angular.module('hunt.appSubmit', ['hunt.backlog'])
     });
   };
 
-  $scope.removeAppSubmit = function () {
-
+  $scope.removeAppSubmit = function (appSubmit, index) {
+    if (window.confirm("Are you sure you want to remove this item from this stage?")){
+      $scope.clickedAppSubmit(appSubmit, index);
```

Comment:
Is there a reason we're calling this function? Do we need rootScope to have access to this object? in selectedAppSubmit?